mimic/canned_responses/loadbalancer.py

```diff
@@ -5,9 +5,7 @@
 """
 from random import randrange
 from copy import deepcopy
-from mimic.util.helper import (EMPTY_RESPONSE,
-                               not_found_response, invalid_resource,
-                               set_resource_status, seconds_to_timestamp)
+from mimic.util.helper import (set_resource_status, seconds_to_timestamp)
```

Comment:
FYI, the parentheses here are unnecessary unless the list of imports is long enough to wrap lines.